src/App.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Books from './components/books';
+import NavBar from './components/nav';
+import Category from './components/categories';
+
+function RenderApp() {
+  return (
+    <>
+      <Router>
+        <NavBar />
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/category" element={<Category />} />
```

Comment:
- [ ] Please, consider adding `exact` for the Books path (`exact path="/"`), as it is required in the project. By adding the `exact` you will consider not showing a blank page when visiting the site for the first time, the Books component should be the default page